src/main/java/uk/gov/hmcts/reform/sscs/service/DeadLetterService.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import com.azure.messaging.servicebus.ServiceBusClientBuilder;
+import com.azure.messaging.servicebus.ServiceBusMessage;
+import com.azure.messaging.servicebus.ServiceBusSenderClient;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.model.DeadLetter;
+
+@Slf4j
+@Service
+public final class DeadLetterService {
```

Comment:
Do we need tests for this actually? I would say not.

Reply:
I think it would be better to integration test this as part of the failed API calls when that part is complete